py3status/modules/stopwatch.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+"""
+A simple stopwatch.
+
+This is a very basic stopwatch. You can start, pause and reset the
+stopwatch.
+
+Button 1 starts/pauses the stopwatch.
+Button 2 resets stopwatch.
+
+Configuration parameters:
+    format: display format for this module (default 'Stopwatch {stopwatch}')
+
+Format placeholders:
+    {stopwatch} display hours:minutes:seconds
+
+@author Jonas Heinrich
+
+SAMPLE OUTPUT
+{'full_text': 'Stopwatch 0:01:00'}
+
+running
+[
+    {'full_text': 'Stopwatch '},
+    {'color': '#00FF00', 'full_text': '0'},
+    {'full_text': ':'},
+    {'color': '#00FF00', 'full_text': '00'},
+    {'full_text': ':'},
+    {'color': '#00FF00', 'full_text': '54'},
+]
+
+paused
+[
+    {'full_text': 'Stopwatch '},
+    {'color': '#FFFF00', 'full_text': '0'},
+    {'full_text': ':'},
+    {'color': '#FFFF00', 'full_text': '00'},
+    {'full_text': ':'},
+    {'color': '#FFFF00', 'full_text': '54'},
+]
+"""
+
+from time import time
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    button_reset = 3
+    button_toggle = 1
+    format = 'Stopwatch {stopwatch}'
+
+    def _reset_time(self):
+        self.running = False
+        self.paused = False
+        self.time_state = None
+        self.color = None
+
+    def post_config_hook(self):
+        self.time_start = None
+        self._reset_time()
+
+    def stopwatch(self):
+
```

Comment:
Kill this line... And keep `def post_config_hook` at top for consistency with other modules.
* `post_config_hook`
* `_reset_time`
* `stopwatch`
* `on_click`

Reply:
okay done